docs/modules/user-guide/partials/ref_devfile-samples.adoc

```diff
@@ -3,5 +3,6 @@
 
 .Additional resources
 
-* link:https://github.com/devfile/api/issues[Creating devfiles].
-* link:https://github.com/devfile/docs[Documenting devfiles].
+To build and test devfiles, visit the link:https://github.com/devfile/api[GitHub API repository]. To see the creation of devfiles, review the open issues. 
```

Comment:
I am not sure if the description is entirely correct. 
The main repo that we have https://github.com/devfile/api is our main repo for source code on the devfile specification as well as tracking issues for all the devfile components. 

Also, if the user has bugs or features that they need to track, they can use https://github.com/devfile/api/issue.

Reply:
Okay, Elson, I'll address this concerns when we get back from the weekend. Thanks for the insight!